optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/statistic/StatisticPoint.java

```diff
@@ -17,66 +17,62 @@
 package org.optaplanner.benchmark.impl.statistic;
 
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.LongStream;
+import java.util.stream.Stream;
 
 /**
  * Implementations must be immutable.
  */
 public abstract class StatisticPoint {
```

Comment:
I haven't seen the coverage data (we will have to wait for them), but I suspect this is one of the less tested classes. Maybe this is a good opportunity to not only modernize the code but also improve its coverage.